psi4/src/psi4/detci/CMakeLists.txt

```diff
@@ -46,6 +46,18 @@ set_source_files_properties(
     COMPILE_FLAGS -O3
   )
 psi4_add_module(bin detci sources)
+
+if(TARGET gauxc::gauxc)
+  target_include_directories(detci
+    PUBLIC
+      $<TARGET_PROPERTY:gauxc::gauxc,INTERFACE_INCLUDE_DIRECTORIES>
+    )
+  target_link_libraries(detci
+    PUBLIC 
+      gauxc::gauxc 
+    )
+endif()
```

Comment:
Why do detci, dfmp2, fisapt, etc. need to include and link gauxc?

Reply:
A lot of the GauXC headers that are used are included in `SplitJK.h`, which in turn get included into `jk.h`, which ends up being included in files in the aforementioned dirs somewhere. Because of that, without these inclusions/links, compilation will fail.

Reply:
Just checked, and you're right that `t_l_l` is unnecessary. I removed `t_l_l` from all of the non-`libfock` `CMakeLists.txt` files where GauXC is present. Thanks!